core/client/fs/src/main/java/alluxio/client/file/cache/CacheManager.java

```diff
@@ -208,6 +209,16 @@ default int get(PageId pageId, int pageOffset, int bytesToRead, byte[] buffer,
   int get(PageId pageId, int pageOffset, int bytesToRead, byte[] buffer, int offsetInBuffer,
       CacheContext cacheContext);
 
+  /**
+   * Get page ids by the given file id.
+   * @param fileId file identifier
+   * @param fileLength file length
+   * @return a list of page ids which belongs to the file
+   */
+  default List<PageId> getPageIdsByFileId(String fileId, long fileLength) {
```

Comment:
can you annotate somewhere in Javadoc, e.g. stating that fileLength will not be needed after we have per-file metadata?

Reply:
done